README.md

```diff
@@ -7,7 +7,7 @@
 
 ## Documentation
 
-Reference documentation can be found at [godoc.org](https://godoc.org/gopkg.in/auth0.v1). For more information about [Auth0](http://auth0.com/) please visit the [Auth0 Docs](http://docs.auth0.com/) page.
+Reference documentation can be found at [godoc.org](https://godoc.org/gopkg.in/auth0.v5). For more information about [Auth0](http://auth0.com/) please visit the [Auth0 Docs](http://docs.auth0.com/) page.
```

Comment:
Should we perhaps update the link to point at https://pkg.go.dev/gopkg.in/auth0.v5 ? godoc.org now redirects there after landing anyway, we could save a hop.